tests/e2e/Services/Realtime/RealtimeBase.php

```diff
@@ -925,6 +925,9 @@ public function testChannelExecutions()
         $this->assertEquals($response['headers']['status-code'], 200);
         $this->assertNotEmpty($response['body']['$id']);
 
+        // Wait for tag to be built.
+        sleep(5);
```

Comment:
I maybe misunderstand, but why are we always waiting for 5s? I cant see `autoDeploy`being used. This has to fail, right?